### PR TITLE
Feature hardware

### PR DESCRIPTION
**Pull request by fvv-vasquez**
Hardware classes - Cellphone and Laptop

### CODE REVIEW

File: sql/bookstore_db.sql [hunk 1]
```diff
@@ -31,7 +31,7 @@ CREATE TABLE `tb_book` (
   `book_genre` varchar(255) NOT NULL,
   `book_modification_date` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
   PRIMARY KEY (`book_id`)
-) ENGINE=InnoDB AUTO_INCREMENT=140 DEFAULT CHARSET=utf8;
+) ENGINE=InnoDB AUTO_INCREMENT=147 DEFAULT CHARSET=utf8;
```
Comment (cssantos0) [inline sql/bookstore_db.sql]: set the  auto increment to 1 in all occurrencies

File: src/main/java/com/fvv/bookstore/bean/Cellphone.java [hunk 2]
```diff
@@ -12,55 +12,31 @@
  */
 public class Cellphone extends Hardware implements Serializable {
 
-	private static final long serialVersionUID = 1L;
-	private Double screenSize;
+	private static final long serialVersionUID = 1L;	
 	private Integer storageMemory;
 	private Integer camera;
```
Comment (cssantos0) [inline src/main/java/com/fvv/bookstore/bean/Cellphone.java]: if this represents pixels, the name should be camPixels

File: src/main/java/com/fvv/bookstore/bean/Laptop.java [hunk 3]
```diff
@@ -14,32 +14,23 @@
...
 	 * @param ramSize of the Laptop
-	 * @param processor of the Laptop
-	 * @param screenSize of the Laptop
 	 * @param hdSize of the Laptop
 	 */
-	public Laptop(final String brand, final String model, final String color, final Double price, 
-			final Integer warranty, final Integer ramSize, final String processor, final Double 
-			screenSize, final Double hdSize) {
-		super(brand, model, color, price, warranty);
+	public Laptop(final String brand, final Double price, final Integer warranty,
```
Comment (cssantos0) [inline src/main/java/com/fvv/bookstore/bean/Laptop.java:29]: your laptop does not have a processor, how are them turn on?

**Review by cssantos0 — CHANGES_REQUESTED** [review]

File: sql/bookstore_db.sql (hunk 1 above)
Comment (fvv-vasquez) [reply sql/bookstore_db.sql]: ok

File: src/main/java/com/fvv/bookstore/bean/Cellphone.java (hunk 2 above)
Comment (fvv-vasquez) [reply src/main/java/com/fvv/bookstore/bean/Cellphone.java]: Ok

File: src/main/java/com/fvv/bookstore/bean/Laptop.java (hunk 3 above)
Comment (fvv-vasquez) [reply src/main/java/com/fvv/bookstore/bean/Laptop.java:29]: ok

**Review by cssantos0 — APPROVED** [review]

**Pull request merged**